kapprover/kapprover.go

```diff
@@ -90,12 +141,14 @@ func tryApprove(filters inspectors.Inspectors, deniers inspectors.Inspectors, wa
 		for _, denier := range deniers {
 			message, err := denier.Inspector.Inspect(client, request)
 			if err != nil {
+				requestsError.WithLabelValues("denyError").Inc()
```

Comment:
Use  the denier.Name as the label value?

Reply:
Done.